File: src/main/kotlin/io/github/maksymilianrozanski/demo/entity/TestTableEntity.java
package io.github.maksymilianrozanski.demo.entity;

import javax.persistence.*;

@Entity
@Table(name="testtable")
public class TestTableEntity {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id")
    private Integer id;

    @Column(name="title")
    private String title;

    @Column(name = "description")
    private String description;


    public TestTableEntity() {
    }

    public TestTableEntity(String title, String description) {
        this.title = title;
        this.description = description;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    @Override
    public String toString() {
        return "TestTableEntity{" + "id=" + id + ", title='" + title + '\'' + ", description='" + description + '\'' + '}';
    }
}

File: src/main/kotlin/io/github/maksymilianrozanski/demo/entity/TestTableEntity.kt
package io.github.maksymilianrozanski.demo.entity

import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.GenerationType
import javax.persistence.Id
import javax.validation.constraints.NotBlank

@Entity(name = "testtable")
data class TestTableEntity(
        @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
        val id: Int = 0,
        @get:NotBlank val title: String = "",
        @get:NotBlank val description: String = ""
)